refactor: specify the events stream replicas

tiagolobocastro opened the pull request.

tiagolobocastro committed:
refactor: specify the events stream replicas

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

niladrih reviewed (commented):
Hi @tiagolobocastro. I don't have context for why this change is required. If I understand this correctly, nats should never have more than 3 replicas?

tiagolobocastro commented:
> Hi @tiagolobocastro. I don't have context for why this change is required. If I understand this correctly, nats should never have more than 3 replicas?

Yes, and it should be set to 1, if there's only 1 replica

tiagolobocastro commented:
bors merge

tiagolobocastro committed:
ci: build repository-local images

The helm chart image CI was not using the images from the PR, this
meant it was using the checked in images!
Now, we build the local images, and ensure we test with them!

Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

tiagolobocastro commented:
bors merge

tiagolobocastro commented:
bors merge

tiagolobocastro commented:
bors try

tiagolobocastro commented:
bors merge
